variables.tf

```diff
@@ -33,6 +33,12 @@ variable "cluster_name" {
 # IAM role
 ################################################################################
 
+variable "platform_role_enabled" {
```

Comment:
Please add a condition in platform_user_enabled that both `platform_role_enabled` and `platform_user_enabled` can't be true